src/index.html

```diff
@@ -17,11 +17,449 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo">
```

Comment:
The `a` tag should have a valid `href` attribute. If you don't have a specific link to put here, you can use `href="#"` as a placeholder.